fe/fe-core/src/main/java/com/starrocks/rpc/BackendServiceProxy.java

```diff
@@ -67,9 +67,14 @@ private BackendServiceProxy() {
         if (Config.enable_brpc_share_channel) {
             rpcOptions.setShareThreadPoolUnderEachProxy(true);
             rpcOptions.setShareChannelPool(true);
-            rpcOptions.setThreadPoolSize(1);
+            rpcOptions.setMaxTotoal(Config.brpc_number_of_concurrent_requests_processed_for_share_channel);
```

Comment:
Will there be a problem when the cluster scale is large?

Reply:
As for async request, because connection can be reused, the connection pool size is not very large, like 8 or 16 is enough.